### ISSUE TITLE
Cooltrainer sprites aren't showing

### ISSUE THREAD

**Issue by BigBahss** [V-FEXrt/porytrainer#2]
Cooltrainer sprites aren't appearing (M or F). I checked a good chunk of the trainers and it seems to to be limited to just cooltrainers.
Additionally, portions of the UI appear cramped (such as the "Double Battle" text under Trainer Properties, the 'Moves' spin boxes, and UI is just slightly cut off by the bottom of the window with the default size . Also some of the 'AI Script Options' texts are elided). I'm sure most of these UI issues are just due to different platforms (I'm on Linux), but I thought I'd point them out. If I get time I may look into these issues myself.

![Screenshot_2020-11-26_03-26-39](https://user-images.githubusercontent.com/58710639/100326250-b6e59b00-2f97-11eb-87eb-26668c85ad37.png)

**Comment by V-FEXrt** [comment]
Thanks for filing!

So the COOLTRAINER is due to a data inconsistency. The key should be COOL_TRAINER to match all the other trainer classes.

Turns out that pret/firered has a similar issue for FARFETCHD. I'm making PRs into both to make the names consistent

**Comment by V-FEXrt** [comment]
After looking into it a bit more, the key name is correct, just needed to change the path to the files.

https://github.com/pret/pokeemerald/pull/1258

**Comment by V-FEXrt** [comment]
- [X] COOLTRAINER image doesn't render. See: pret/pokeemerald#1258
- [X] Farfetchd doesn't render in FRLG. See https://github.com/pret/pokefirered/pull/373
- [X] "Double Battle" text under Trainer Properties cramped
- [ ] 'Moves' spin boxes cramped
- [ ] UI is just slightly cut off by the bottom of the window with the default size
- [ ] Also some of the 'AI Script Options' texts are elided
    - This was intentional, but I can look into actually solving it

**Comment by BigBahss** [comment]
Yeah I briefly looked into it filing this and noticed that inconsistency. I think changing changing the filenames in the decomps is the right solution.

Additionally, I looked at porytrainer on my laptop that has a lower res screen and realized that the UI elements can't fit in the window when maximized, should probably have a scroll area somewhere. (In hindsight I should've made a separate issue for the UI related stuff.)

**Comment by V-FEXrt** [comment]
I've pushed up some changes if you could confirm they look okay on your side. Not sure what I am going to do about the spin boxes though, I guess the solution is just to make the window wider.

I'll have to look into the scrolling a bit, I'm not too familiar with the UI tooling yet

**Comment by BigBahss** [comment]
I just saw your comment, my bad!
I looked at the changes, they look good on my end. I can put together a PR to fix the layout problems. Is the intended appearance for the 'Moves' label to be between the spin-boxes and the combo-boxes, the way that 'Level' and 'IV' is?

**Comment by V-FEXrt** [comment]
Yeah, that is exactly the intent! Thanks!

**Comment by V-FEXrt** [comment]
BTW. you'll probably want to update your fork to avoid merge conflicts
